### DISCUSSION TITLE
path to node

### DISCUSSION THREAD

**Discussion by irajgreenberg** [coreybutler/nvm-windows#595] [General] [answered]
I've tried about 10 times now to install npm (Win 10), but still can't find node/npm  post install. I've removed node before the installs. I've tried your setup install and also via choocolatey; same result. Thanks for any assistance.

**Comment by coreybutler** [comment, marked as answer]
You don't need to install npm, the tool does it for you.

Once NVM4W is installed, close the terminal/powershell and re-open it (this only needs to be done once after the install to refresh the environment variables). Then you have to run `nvm install x.x.x` where `x.x.x` is the version you want. Then run `nvm use x.x.x` to use that version.

**Comment by irajgreenberg** [comment]
Thanks!

On Mon, Jan 4, 2021 at 2:01 PM Corey Butler <notifications@github.com>
wrote:

> You don't need to install npm, the tool does it for you.
>
> Once NVM4W is installed, close the terminal/powershell and re-open it
> (this only needs to be done once after the install to refresh the
> environment variables). Then you have to run nvm install x.x.x where x.x.x
> is the version you want. Then run nvm use x.x.x to use that version.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coreybutler/nvm-windows/discussions/595#discussioncomment-261170>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAQNOW72PXID7ZWSEJEYIO3SYIUDTANCNFSM4U6QGZQQ>
> .
>


-- 
Ira Greenberg
Chair and Professor, Division of Art
Director, Center of Creative Computation
SMU, Dallas

214.768.4625

https://www.smu.edu/Meadows/AreasOfStudy/CreativeComputation/Faculty/GreenbergIra

Twitter: @Iragreenberg, @processingTips
Facebook: www.facebook.com/irajgreenberg
Soundcloud: https://soundcloud.com/ira-greenberg-485691189
